src/main/java/crazypants/enderio/TileEntityEio.java

```diff
@@ -32,14 +33,19 @@ protected boolean shouldUpdate() {
     return true;
   }
 
+  private long lastUpdate = 0;
+
   @Override
   public final void updateEntity() {
-    doUpdate();
-    if(isProgressTile && !worldObj.isRemote) {
-      int curScaled = getProgressScaled(16);
-      if(++ticksSinceLastProgressUpdate >= getProgressUpdateFreq() || curScaled != lastProgressScaled) {
-        sendTaskProgressPacket();
-        lastProgressScaled = curScaled;
+    if (Config.allowExternalTickSpeedup || worldObj == null || worldObj.getTotalWorldTime() != lastUpdate) {
```

Comment:
@Vexatos It means that it will not execute the method call on a null value but continue.

@MatthiasMann About half the doUpdate()s start with a null check. Hard to not type it after having seen it that often...

I'll clean them up later https://github.com/HenryLoenwind/EnderIO/issues/20